Core/src/main/java/com/plotsquared/core/plot/flag/types/BlockTypeWrapper.java

```diff
@@ -44,16 +46,25 @@
 public class BlockTypeWrapper {
```

Comment:
Remove equalsAndHashCode and implement it using equals and hashcode builders from guava/apache (intellij has a generation thingy for this :cow:)